content/2018-munich/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+---
+
+## Overview
+
+PromCon 2018 is the third conference dedicated to the
+[Prometheus monitoring system](https://prometheus.io/). It will take place
+2018-08-09 & 2018-08-10 (Thu & Fri)
```

Comment:
While we all know that this is the one and true date format, when interfacing with normal humans I would prefer "August 9 - 10" over this.

Reply:
You seem to imply that our target audience are normal humans ;)

Can change, though.